merkledag/utils/utils.go

```diff
@@ -153,3 +148,32 @@ func rmLink(ctx context.Context, ds dag.DAGService, root *dag.Node, path []strin
 
 	return root, nil
 }
+
+func (e *Editor) WriteOutputTo(ds dag.DAGService) error {
+	return copyDag(e.GetNode(), e.ds, ds)
+}
```

Comment:
this doesn't seem to get called? shouldn't it be called by the add command at the end?